os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -200,11 +200,12 @@ static void up_taskdump(FAR struct tcb_s *tcb, FAR void *arg)
 	size_t used_stack_size;
 
 	if (tcb->pid == 0) {
-		used_stack_size = 0;
-	} else {
-		used_stack_size = up_check_tcbstack(tcb);
+		tcb->adj_stack_size = CONFIG_IDLETHREAD_STACKSIZE;
```

Comment:
If any of these 2 config is not set, then?

Reply:
CONFIG_IDLETHREAD_STACKSIZE is mandantory. It should be set.

Reply:
Got it.